pandas/tests/generic/test_finalize.py

```diff
@@ -784,3 +783,28 @@ def test_finalize_frame_series_name():
     df = pd.DataFrame({"name": [1, 2]})
     result = pd.Series([1, 2]).__finalize__(df)
     assert result.name is None
+
+
+def test_concat_retain_attrs():
```

Comment:
I suppose that these tests have to be placed in ``pandas/tests/reshape/concat/test_concat.py``.
Is it correct?